doc/ref/tops/all/index.rst

```diff
@@ -15,4 +15,8 @@ master tops modules
     mongo
     reclass_adapter
     saltclass
+<<<<<<< HEAD
     varstack
+=======
+    varstack_top
+>>>>>>> merge-fluorine
```

Comment:
there is an extra merge conflict here.

Reply:
d'oh

Reply:
Force-pushed a fixup commit to take care of this.